recipe/meta.yaml

```diff
@@ -14,17 +14,17 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.7
     - pip
 
   run:
-    - python >=3.8
+    - python >=3.7
     - pip
     - coiled
     - nodejs ==17.8.0
     - nb_conda_kernels ==2.3.1
     - numpy ==1.21.5
-    - pandas ==1.4.1
+    - pandas ==1.3.5
```

Comment:
Python 3.7 was dropped starting with `pandas=1.4.0`. `1.3.5` is the last release with 3.7 support

Reply:
Yeah, I guess we have to do this until we drop Python 3.7 support. 😦